docs/administration-guide/gaffer-deployment/gaffer-docker/how-to-run.md

```diff
@@ -0,0 +1,270 @@
+# How to Deploy with Accumulo on Docker
+
+After reading the [previous page](./gaffer-images.md) you should have a good
+understanding of what images are available for Gaffer and how to configure them
+to you needs. However, before running a deployment backed by Accumulo you will
+need to know a bit of background on Hadoop to understand how the data will scale
+and be distributed.
+
+Usually when deploying a container image you simply run the image and everything
+is contained locally to the container (hence the name). For larger scale graphs
+this less desireable as we will usually want to be able to scale and load
+balance the storage based on the volume of data; this is where Hadoop comes in.
+
+## What is Hadoop/Accumulo?
+
+[Apache Hadoop](https://hadoop.apache.org/) is an open-source software framework
+used for distributed storage and processing of large datasets. Hadoop is
+designed to handle various types of data, including structured, semi-structured,
+and unstructured data. It is a highly scalable framework that allows users to
+add nodes to the cluster as needed.
+
+Hadoop has two main components: Hadoop Distributed File System (HDFS) and
+MapReduce. HDFS is a distributed file system that provides high-throughput
+access to data. MapReduce is a programming model used for processing large
+datasets in parallel.
+
+Accumulo is built on top of the HDFS to provide a key-value store with all the
+same scalability and robustness of Hadoop.
+
+## Running a Cluster via Docker for Gaffer
```

Comment:
This is manually walking through the steps which occur when running the docker compose file. I think it would be good to make that clear and link to that file.

Reply:
Agreed probably should add a link in, yeah the idea was that using standard docker means should be translatable to other tools other than docker compose